tests/testtools/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2025 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET aostesttools)
```

Comment:
Probably not so right place for testtools, but can't come up with a better one. Let's find out when move to mono repo.